molo/yourwords/admin.py

```diff
@@ -0,0 +1,18 @@
+from molo.core.models import ArticlePage
+from molo.yourwords.models import YourWordsCompetitionEntry
+from django.contrib import admin
+
+
+def ConvertToArticle(model_admin, request, entry):
+    article = ArticlePage.objects.create(null, null)
```

Comment:
Just another note that creating an article is not enough, you'll need to save it as a draft which looks something like this
```
article.save_revision(
    user=request.user,
    submitted_for_moderation=False,
)
```